#BE-4886 Add fromDisplayName info for Testing Distribution emails (#744)
appcircleio/appcircle-docusaurus: docs/self-hosted-appcircle/configure-server/integrations-and-access/integration.md

The `verifyCertificate` option is available in version `3.23.1` or later.
:::

:::info

> The `fromDisplayName` is used as the email sender name for email notifications. However, when you [share a binary](https://docs.appcircle.io/testing-distribution/create-or-select-a-distribution-profile#share-binary) using the Testing Distribution module, the **profile name** will be displayed as the sender, as shown in the example email provided in the linked documentation.

Reviewer: Using active voice improves clarity. Also, add "from name" as an alternative naming for the "sender name".

```suggestion
Email notifications use the `fromDisplayName` as the sender name (or from name). However, when you [share a binary](https://docs.appcircle.io/testing-distribution/create-or-select-a-distribution-profile#share-binary) using the **Testing Distribution** module, the **profile name** will be displayed as the sender, as shown in the example email provided in the linked documentation.
```

Using the full URL here is OK since we have scroll problems with the anchor (`#`) URLs. ✅
